### DISCUSSION TITLE
Using takeout on digital ocean

### DISCUSSION THREAD

**Discussion by nezaboravi** [tighten/takeout#232] [Q&A] [answered]
Hi, wanna ask if is possible to use takeout on digital ocean?
Somehow to enable it via laravel forge or something like that?
I need it cause of meilisearch and before i deep dive into it, want to know if this is limited to local development only?

**Comment by mattstauffer** [comment, marked as answer]
Our plan for it is to only ensure it works for local development. I'd probably recommend against it, mainly because none of our settings are planned for public usage, so the ports, passwords, and other configurations are set for insecure local development.
